packages/contracts/remappings.txt

```diff
@@ -4,3 +4,4 @@ ds-test/=lib/forge-std/lib/ds-test/src/
 solidity-linked-list/=lib/solidity-linked-list
 @uniswap/v2-core/contracts/=lib/Uniswap/v2-core/contracts
 @uniswap/v2-periphery/contracts/=lib/Uniswap/v2-periphery/contracts
+solidity-stringutils/=lib/solidity-stringutils/src/
```

Comment:
> same what I mentioned above, we're gonna remove library dependencies as much as possible which are not openzeppelin ones.

Yeah, will not use openzeppelin, it was needed for dealing with original manager contract on our diamond

Reply:
> same what I mentioned above, we're gonna remove library dependencies as much as possible which are not openzeppelin ones.

You mean, we can keep using Openzeppelin libraries and not want to use any others like string utils?

Reply:
I say we should be using openzeppelin even in unit test. I am 100% sure we can do everything what solidity-stringutils.

Reply:
Solidity-stringutils is useful for forge test because it makes manipulation of dumping data as string in solidity, I think
@0xcodercrane if you know alternatives for this on OZ, please let me know it

Reply:
Right now, you used it for generating function selectors for facets from its name.  I was saying we can use another way like DiamondLoupeFacet.facets.selector rather than using `forge inspect FACET_NAME methods` but yeah we can skip bc its not a high priority.